src/gbt_model_builder.pyx

```diff
@@ -179,3 +179,33 @@ def gbt_reg_model_builder(n_features, n_iterations):
     :param size_t n_iterations: Number of trees in the model
     '''
     return gbt_regression_model_builder(n_features, n_iterations)
+
+class GBTModel():
```

Comment:
Should be derived from sklearn BaseEstimator. Since sklearn is runtime dependency, this class should not be compiled during build

```suggestion
class GBTModel(BaseEstimator):
```

Reply:
done